### ISSUE TITLE
Apache : syslog logging method breaks rules

### ISSUE THREAD

**Issue by benrubson** [fail2ban/fail2ban#1693]
Hello,

Apache can easily log using syslog :
`ErrorLog syslog`

Unfortunately this breaks fail2ban rules.

Here is an example of Apache syslog lines :
```
Feb 15 16:23:00 srv httpd[22034]: [authz_core:error] [pid 22034] [client 1.2.3.4:58585] AH01630: client denied by server configuration: /home/www/
Feb 15 16:23:40 srv httpd/backend1[22034]: [authz_core:error] [pid 22036] [client 1.2.3.4:59392] AH01630: client denied by server configuration: /home/backend1/
Feb 15 16:54:53 tools apache2[18154]: [:error] [pid 18154:tid 140680873617152] [client 1.2.3.4:48154] AH01630: client denied by server configuration: /var/www/
```

Sounds like the following would need to be extended ?
```
_apache_error_client = \[\] \[(:?error|\S+:\S+)\]( \[pid \d+(:\S+ \d+)?\])? \[client <HOST>(:\d{1,5})?\]
```

Thank you very much 👍 

Best regards,

Ben

**Comment by benrubson** [comment]
Solved by #1695 !

**Comment by benrubson** [comment]
Closed, see PR.
